src/main/java/com/alkemy/ong/controller/CategoryController.java

```diff
@@ -0,0 +1,29 @@
+package com.alkemy.ong.controller;
+
+import com.alkemy.ong.entity.CategoryEntity;
+import com.alkemy.ong.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/Category")
```

Comment:
I think the routes should start lower case, this way /category?

Reply:
you are right the category is in lowercase

Reply:
now i make the change

Reply:
Maybe just /categories without the first /category, I think in the tickets only mentions categories, simpler

Reply:
hecho en la resolución de conflicto